docs/DeveloperGuide.md

```diff
@@ -1057,6 +1065,10 @@ testers are expected to do more *exploratory* testing.
    Expected: 2nd student is edited. Details of the edited student is shown.
 4. Test case: `edit 2 n/Test 2`<br>
    Expected: No student is edited. Error details shown.
+5. Test case: `edit 1 c/CS2103T;CS2109S`
```

Comment:
Can make it more explicitly that the student's previous courses are removed and are replaced by both CS2103T and CS2109S.